app/tests/spec/views/sign_up.js

```diff
@@ -79,6 +84,20 @@ define(function (require, exports, module) {
       assert.equal(args[0], expectedMessage);
     }
 
+    function enableExperiments() {
+      var windowMock = new WindowMock();
```

Comment:
It's alright to introduce `const` and other ES2015 feature as you modify a module. No need to use all the old stuff just because the original tests were written that way.